runtime/near-vm-runner/src/wasmer2_runner.rs

```diff
@@ -24,6 +25,21 @@ use wasmer_compiler_singlepass::Singlepass;
 use wasmer_types::InstanceConfig;
 use wasmer_vm::TrapCode;
 
+const WASMER_FEATURES: wasmer::Features = wasmer::Features {
+    threads: WASM_FEATURES.threads,
+    reference_types: WASM_FEATURES.reference_types,
+    simd: WASM_FEATURES.simd,
+    bulk_memory: WASM_FEATURES.bulk_memory,
+    multi_value: WASM_FEATURES.multi_value,
+    tail_call: WASM_FEATURES.tail_call,
+    module_linking: WASM_FEATURES.module_linking,
+    multi_memory: WASM_FEATURES.multi_memory,
+    memory64: WASM_FEATURES.memory64,
+    exceptions: WASM_FEATURES.exceptions,
+    // singlepass does not support signals.
+    signal_less: true,
```

Comment:
unreated to PR: looking at this, I wonder if maybe we should place the `signal_less` toggle into some other config? signals are not a wasm feature, they are an impl detal of our version of wasmer. 

Basically, by default wasmer handles traps by relying on signals, and that was observed to cause troubles (https://github.com/near/nearcore/blob/3ed678195025c643ecb6711c577545bc138b37ea/runtime/near-vm-runner/src/wasmer_runner.rs#L86-L118 beinnd manifestation of that). 

So @olonho changed singlepass to *not* use signals, and that requried a flag to tell wasmer runtime (which is shared between backends) to not install signal handlers.